tidb-community-upgrade.md

```diff
@@ -0,0 +1,130 @@
+---
+title: New Structure, New Roles - TiDB Community Upgrade!
+author: ['Jian Zhang']
+date: 2019-10-30
+summary: Four years of development has seen the TiDB community gradually grow into maturity. However, as the community expands, some shortcomings begin to loom out of the current community structure. After a series of reflections and summaries, we decided to upgrade and adjust the current community organization structure, with more community roles and community organizations introduced to better stimulate the vitality of the community and maintain a positive and healthy community environment.
```

Comment:
```suggestion
summary: PingCAP has upgraded and adjusted the TiDB community organization structure, with more community roles and community organizations introduced to better stimulate the vitality of the community and maintain a positive and healthy community environment.
```

Reply:
I think this summary can be shorter so that it can be displayed completely in the link preview.